pennylane_lightning/core/src/bindings/Bindings.hpp

```diff
@@ -172,20 +173,23 @@ auto alignedNumpyArray(CPUMemoryModel memory_model, size_t size) -> py::array {
  * @param size Size of the array to create
  * @param dt Pybind11's datatype object
  */
-auto allocateAlignedArray(size_t size, const py::dtype &dt) -> py::array {
+auto allocateAlignedArray(size_t size, const py::dtype &dt,
```

Comment:
What do you think of adding `std::memset` to this method to remove `zeroInit` argument with some default value from the rest of C++ codebase? I assume `zeroInit` is mainly useful in this method and removing it from other places would reduce the number of code changes while keeping the signature of those methods simple.

Reply:
Happy to in a follow up PR.

Reply:
TODO added.